libsignal-service-dotnet/websocket/ISignalWebSocket.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace libsignalservicedotnet.websocket
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+    public class SignalWebSocketMessageReceivedEventArgs
+    {
+        public Stream Message { get; set; }
+    }
+
+    public interface ISignalWebSocket : IDisposable

```

Comment:
What is actually implementing this?

Reply:
@golf1052  Every app that uses this lib will have to.  Signal-Windows will have a reference implementation.  I was commenting above about the possibility of putting a default implementation in the lib (although to make it .net core relevant we would still need the dual build package).